administrator/components/com_joomgallery/helpers/html/joomconfig.php

```diff
@@ -129,4 +129,353 @@ public static function row($key, $type, $name, $info, $display = true, $attribs
+
+        <div id="testsettings" class="accordion">
+          <div class="accordion-group">
+            <div class="accordion-heading">
+              <p id="detailIMG_title" class="accordion-title">
+                <?php echo JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_DETAILINFO'); ?>
+              </p>
+            </div>
+            <div id="detailIMG" class="accordion-body">
+              <div class="accordion-inner">
+                <table class="adminlist table table-bordered">
+                  <tbody>
+                    <tr>
+                      <td valign="top" align="left">
+                        <button id="button" class="btn" type="detail" side="L" onclick="submitGenerate(event)"><?php echo JText::_('COM_JOOMGALLERY_COMMON_TOOLBAR_RECREATE'); ?></button>
+                        <br /><br />
+                        <?php if(!empty($info->detailL)): ?>
+                          <p><strong <?php echo JHTML::_('joomgallery.tip', JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_FILESIZE_TXT'), JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_FILESIZE_LBL'), true); ?>><?php echo JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_FILESIZE_LBL');?>:</strong> <?php echo round(($info->detailL->filesize) / 1024, 0);?> kB</p>
+                          <p><strong <?php echo JHTML::_('joomgallery.tip', JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_FILETYPE_TXT'), JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_FILETYPE_LBL'), true); ?>><?php echo JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_FILETYPE_LBL');?>:</strong> <?php echo $info->detailL->filetype;?></p>
+                          <p><strong <?php echo JHTML::_('joomgallery.tip', JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_DIMENSION_TXT'), JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_DIMENSION_LBL'), true); ?>><?php echo JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_DIMENSION_LBL');?>:</strong> <?php echo $info->detailL->dimension;?></p>
+                          <p><strong <?php echo JHTML::_('joomgallery.tip', JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_FRAMES_TXT'), JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_FRAMES_LBL'), true); ?>><?php echo JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_FRAMES_LBL');?>:</strong> <?php echo $info->detailL->frames;?></p>
+                          <p><strong <?php echo JHTML::_('joomgallery.tip', JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_GENTIME_TXT'), JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_GENTIME_LBL'), true); ?>><?php echo JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_GENTIME_LBL');?>:</strong> <?php echo round($info->detailL->processing_time,3);?> sec</p>
+                          <p><strong <?php echo JHTML::_('joomgallery.tip', JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_GENMEMORY_TXT'), JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_GENMEMORY_LBL'), true); ?>><?php echo JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_GENMEMORY_LBL');?>:</strong> <?php echo round(($info->detailL->used_memory) / pow(1000,2),2);?> MB</p>
+                        <?php else: ?>
```

Comment:
Add
`                          <p><strong <?php echo JHTML::_('joomgallery.tip', JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_QUALITY_TXT'), JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_QUALITY_LBL'), true); ?>><?php echo JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_QUALITY');?>:</strong> <?php echo ($info->detailL->used_quality);?></p>`
to show the quality parameter